tests/Unit/Domain/Test_Domain.cpp

```diff
@@ -1161,9 +1162,13 @@ Domain<3> create_serialized_domain() {
   Maps maps{};
 
   const Translation translation_A{
-      Affine{-1.0, 1.0, -1.0 + x_coord_a, 1.0 + x_coord_a}, Identity2D{}};
+      Affine{-1.0, 1.0, -1.0 + x_coord_a, 1.0 + x_coord_a},
+      Affine{-1.0, 1.0, -1.0 + y_offset, 1.0 + y_offset},
+      Affine{-1.0, 1.0, -1.0 + z_offset, 1.0 + z_offset}};
```

Comment:
I think for the test to succeed you have to drop this change, and the one for `translation_B` below. Because these are the maps actually written to disk in the old dataset.

Reply:
That worked, thanks!